e2e/nomostest/ntopts/multi_repo.go

```diff
@@ -49,6 +49,9 @@ type MultiRepo struct {
 	// SkipNonLocalGitProvider will skip the test if run with a GitProvider type other than local.
 	SkipNonLocalGitProvider bool
 
+	// SkipNonLocalRegistryProvider will skip the test if run with a RegistryProvider type other than local.
+	SkipNonLocalRegistryProvider bool
```

Comment:
Kinda wish these were just test requirements and not a double negative Skip- Non-, but there's precedent, so i guess it's ok.

Reply:
They could be renamed to `RequireLocal*Provider`. I suggest we refactor that separately

Reply:
https://github.com/GoogleContainerTools/kpt-config-sync/pull/1109